modules/core/src/test/scala/fs2/kafka/KafkaAdminClientSpec.scala

```diff
@@ -180,27 +180,18 @@ final class KafkaAdminClientSpec extends BaseKafkaSpec {
           .resource[IO](adminClientSettings)
           .use { adminClient =>
             for {
-              topicNames <- adminClient.listTopics.names.map(
-                _.filterNot(_.startsWith("__confluent"))
-              )
-              _ <- IO(assert(topicNames.size == 1))
-              topicListings <- adminClient.listTopics.listings.map(
-                _.filterNot(_.name.startsWith("__confluent"))
-              )
-              _ <- IO(assert(topicListings.size == 1))
-              topicNamesToListings <- adminClient.listTopics.namesToListings.map(_.filter {
-                case (key, _) => !key.startsWith("__confluent")
-              })
-              _ <- IO(assert(topicNamesToListings.size == 1))
+              topicNames <- adminClient.listTopics.names
+              topicCount = topicNames.size
```

Comment:
Instead of having a fixed expectation of how many topics there should be, we check how many there are at the start and then generate other expectations relative to that.